maven-java/pom.xml

```diff
@@ -46,34 +46,89 @@
   </repositories>
 
   <distributionManagement>
-    <repository>
-      <id>lightbend-akkaserverless-cloudsmith</id>
-      <name>Cloudsmith</name>
-      <url>https://maven.cloudsmith.io/lightbend/akkaserverless/</url>
-    </repository>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
   </distributionManagement>
 
   <build>
     <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.sonatype.plugins</groupId>
+          <artifactId>nexus-staging-maven-plugin</artifactId>
+          <version>1.6.7</version>
+	  <extensions>true</extensions>
+          <configuration>
+            <serverId>ossrh</serverId>
+            <nexusUrl>https://s01.oss.sonatype.org</nexusUrl>
+            <autoReleaseAfterClose>false</autoReleaseAfterClose>
```

Comment:
we could set this to 'true' eventually, but for the first release perhaps let's give ourselves a chance to check the artifacts?